tests/micro_benchmarks/benchmarks.sh

```
@@ -0,0 +1,179 @@
+    exit 1
+fi
+
+if [[ $LIST == 1 ]]; then
+    BENCH_ARGS+=" --list"
+fi
+
+E=0
+
+$READIES/bin/sep
+echo "# Running micro benchmarks"
+BENCHS_DIR="$(cd $BINROOT/src/tests/micro_benchmarks; pwd)"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)